components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -128,27 +146,58 @@ export default function () {
-                                    Unpinned workspaces that have been inactive for more than 14 days will be
-                                    automatically deleted.{" "}
-                                    <a
-                                        className="gp-link"
-                                        href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection"
+                                <div className="pt-14">
+                                    <div className="border-t border-gray-200 dark:border-gray-800"></div>
+                                    <div
+                                        onClick={() => setShowInactive(!showInactive)}
+                                        className="flex cursor-pointer py-6"
                                     >
-                                        Learn more
-                                    </a>
+                                        <div className="flex flex-col">
+                                            <h2 className="">Inactive Workspaces</h2>
+                                        </div>
+                                        <div>
+                                            <Arrow
+                                                up={!!showInactive}
+                                                customBorderClasses="text-gray-400 dark:text-gray-500 hover:text-gray-600 dark:hover:text-gray-400"
+                                            />
+                                        </div>
+                                    </div>
+                                    {showInactive ? (
+                                        <>
+                                            <div className="flex flex-row p-3 text-gray-400 bg-gray-50 dark:bg-gray-800 rounded-xl">
+                                                <div className="text-sm flex-auto py-3">
+                                                    Unpinned workspaces that have been inactive for more than 14 days
+                                                    will be automatically deleted.{" "}
+                                                    <a
+                                                        className="gp-link"
+                                                        href="https://www.gitpod.io/docs/life-of-workspace/#garbage-collection"
+                                                    >
+                                                        Learn more
+                                                    </a>
+                                                </div>
+                                                <button
+                                                    onClick={() => setDeleteModalVisible(true)}
+                                                    className="ml-2 danger secondary w-64 p-2"
+                                                >
+                                                    Delete Inactive Workspaces
```

Comment:
**suggestion**: Using the new collapsible element described above instead of the info alert component, could allow use to also be more flexible with other elements like the action button here.

| Link button | Normal button |
|-|-|
| <img width="1440" alt="InactiveWorkspaces-link" src="https://user-images.githubusercontent.com/120486/172181490-92677e25-fcef-419a-9675-6a97feb8f98b.png"> | <img width="1440" alt="InactiveWorkspaces-button" src="https://user-images.githubusercontent.com/120486/172181494-7265b4c2-0717-4d5f-be55-ee1a920dd218.png"> |

Reply:
Marking as resolved. Moved forward with a secondary danger button. ✔️